manifest-list.md

```diff
@@ -36,7 +38,7 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 
   - **`platform`** *object*
 
-    This REQUIRED property describes the platform which the image in the manifest runs on.
+    This OPTIONAL property describes the platform which the image in the manifest runs on.
```

Comment:
For those cases, it will be present. There is a lot of discussion above, but the idea is that there might be companion data that is _not_ platform-specific.

@vbatts We may want to bolster this with something saying "This property SHOULD be present if its target is platform-specific." Maybe, even a must?